__tests__/app.test.js

```diff
@@ -59,4 +60,49 @@ describe('app()', () => {
         });
     });
   });
+  describe('/api/articles/:article_id', () => {
+    it('200: return with a status of 200', () => {
+      return request(app).get('/api/articles/1').expect(200);
+    });
+    it('200: return with a status of 200 and the corresponding article 1 data', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toHaveProperty('author', expect.any(String));
+          expect(article).toHaveProperty('title', expect.any(String));
+          expect(article).toHaveProperty('article_id', expect.any(Number));
+          expect(article).toHaveProperty('body', expect.any(String));
+          expect(article).toHaveProperty('topic', expect.any(String));
+          expect(article).toHaveProperty('created_at', expect.any(String));
+          expect(article).toHaveProperty('votes', expect.any(Number));
+          expect(article).toHaveProperty('article_img_url', expect.any(String));
+        });
+    });
```

Comment:
This is looking great, when we're checking for a specific object like we are here we can just use the actual object to check for equality, however any time we loop we will want to use this method.